### ISSUE TITLE
DIORA experimental settings

### ISSUE THREAD

**Issue by jxjessieli** [i-lijun/UnsupConstParseEval#2]
Hi, I noticed that DIORA used a larger dataset from MultiNLI and SNLI to train the model. While you used PTB and KTB to train the model. May I know if you used exactly the same hyperparameters from the original DIORA to get the results? If not, what are the hyperparameters used in your setting? 
Thank you!

**Comment by i-lijun** [comment]
Hi @jessiexiye 

Thanks for your interest. We use the same hyperparameters as in the [original paper](https://github.com/iesl/diora#quick-start), except that the learning rate is 1e-3 rather than 2e-3.
